component/integration/aws/sqs/alarms.go

```diff
@@ -125,9 +127,7 @@ func (a alert) toView() (MessageView, error) {
 		}
 	}
 
-	v := MessageView{
-		ID: fmt.Sprintf("%s-%s", name, alias),
-	}
-
-	return v, nil
+	return MessageView{
+		ID: fmt.Sprintf("arn:aws:lambda:us-east-1:%s:function:%s:%s", msg.AccountID, name, alias),
```

Comment:
I think hard coding the region might bring some issues in the case the lambda is hosted on a different region

Reply:
The region could be parsed from the CloudWatch alarm ARN since the event JSON does not contain the Lambda function region due to the detached nature of custom alarm metrics. However, this could be the incorrect region since the Lambda function triggering the alarm may or may not be in the same region that the alarm was created in. Not sure of a good way to get the function ARN region from the event in order to created the Lambda ARN. Looking into if the region can be specified when creating the alarm to make sure it is in the same region as the function.

Reply:
Ok, I am playing with the idea of using the Alarms description field to store the Lambda function ARN that triggers it. It is possible that this alarm could be triggered by another Lambda function with the same name and alias in a different region. I have not figured out a way to store or monitor CloudWatch metrics by region. I know the metrics are stored in a specific region, but not sure if the metrics are aggravated by the resource's region. I do not see region as an option in the dimensions list for metrics.

Reply:
It may be acceptable to flag any lambda card that represent the same lambda name and alias regardless of region in the portal when any fail. However, it could be confusing since each function's arn can be specified when setting up the function and that includes the region.

Reply:
In the future, it may be needed to explore creating the alarm name using the lambda region.